medium-tests/src/test/java/mediumtest/tracking/IssueTrackingMediumTests.java

```diff
@@ -713,6 +717,55 @@ void it_should_match_a_previously_tracked_issue_with_a_server_issue_when_binding
       .containsExactly(firstPublishedIssue.getId(), "key", serverIssueIntroductionDate);
   }
 
+  @Test
+  void it_should_submit_server_path_to_sc_web_api(@TempDir Path baseDir) {
+    var folder = createFolder(baseDir, "local/path/prefix");
+    var filePath = createFile(baseDir, folder.resolve("pom.xml").toString(),
+      "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n"
+        + "<project>\n"
+        + "  <modelVersion>4.0.0</modelVersion>\n"
+        + "  <groupId>com.foo</groupId>\n"
+        + "  <artifactId>bar</artifactId>\n"
+        + "  <version>${pom.version}</version>\n"
+        + "</project>");
+    var fileUri = filePath.toUri();
+    var orgKey = "myOrganization";
+    var connectionId = "connectionId";
+    var projectKey = "projectKey";
+    var server = newSonarCloudServer(orgKey)
+      .withProject(projectKey, project -> project
+        .withFile("server/path/prefix/pom.xml")
+          .withQualityProfile("qp")
+      )
+      .withQualityProfile("qp", qualityProfile -> qualityProfile.withLanguage("xml")
+        .withActiveRule("xml:S3421", rule -> rule.withSeverity(IssueSeverity.MAJOR)))
+      .withVersion("8.0.0.55884")
```

Comment:
Why do you need a version for SonarCloud?

Reply:
https://github.com/SonarSource/sonarlint-core/blob/55a175cddc96ad0cc015ec8ad13e7c102d7724b3/backend/server-connection/src/main/java/org/sonarsource/sonarlint/core/serverconnection/LocalStorageSynchronizer.java#L54

This fails for the `null` version and blocks all further sync.